src/cdk-experimental/scrolling/virtual-scroll-viewport.html

```diff
@@ -1,11 +1,14 @@
+<!-- Place the ng-content in a template that we can conditionally render in different places. -->
+<ng-template #rawContent><ng-content></ng-content></ng-template>
```

Comment:
Per offline discussion, we can change this to just use straight `<ng-content>` with a selector for `[cdkVirtualScrollContent]`